MagaluCloud/magalu
# feat: profile integration

## Description



Integrate profiles to the project.

The `config` and `auth` packages now read and write to the current profile using the `ProfileManager` instance coming from the `Sdk`.

`BuildMgcPath` and `BuildMgcFilePath` functions were removed. The `ProfileManager` should be the only one able build this path, and read/write from/to it.

## Related Issues


## Progress





### Pull request checklist




## How to test it



- Remove the `./config/mgc` directory
- Running `auth login` should create: `.config/mgc/default/auth.yaml`
- Running `config set --key region --value br-ne-1` should create: `.config/mgc/default/cli.yaml`
- Test other commands. The CLI must behave as before
